engine/core/src/main/java/org/datacleaner/reference/ReferenceDataCatalogImpl.java

```diff
@@ -109,6 +109,17 @@ public Dictionary getDictionary(String name) {
 		return null;
 	}
 
+	private boolean removeItem(Collection collection, String name) {
```

Comment:
Maybe revert  the whole file then. as the only changes would then be the replace-with-diamond changes.